docs/lib/predictions/fragments/android/getting-started.md

```diff
@@ -0,0 +1,171 @@
+    * What is the source language? `English`
+    * What is the target language? `Italian`
+    * Who should have access? `Auth and Guest users`
+
+4. Run `amplify push` to create the resources in the cloud.
+
+When your backend is successfully updated, verify that `amplifyconfiguration.json` and `awsconfiguration.json` are properly updated inside the `res/raw` directory. Your new configuration file `amplifyconfiguration.json` will contain your default project `region` value as well as any necessary configuration options for each predictions method (i.e. target language and source language for translate, etc).
+
+
+## Manual Configuration
+
+If you have already created the resources in the cloud and would like to take advantage of those existing resources but still use the Amplify library in Android Studio, please follow the directions below:
+
+Create the file `amplifyconfiguration.json`:
+```
+touch amplifyconfiguration.json
+```
+
+Copy the contents over and update the values for the specific predictions method you are looking to use:
+```
+{
+    "UserAgent": "aws-amplify-cli/2.0",
+    "Version": "1.0",
+    "Predictions": {
+        "plugins": {
+            "awsPredictionsPlugin": {
+                "defaultRegion": "us-west-2",
+                 "identify": {
+                    "identifyText": {
+                        "format": "ALL",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "identifyEntities": {
+                        "maxEntities": "0",
+                        "celebrityDetectionEnabled": "true",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "identifyLabels": {
+                        "region": "us-west-2",
+                        "type": "LABELS",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                },
+                "convert": {
+                    "translateText": {
+                        "targetLang": "zh",
+                        "sourceLang": "en",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "speechGenerator": {
+                        "voice": "Salli",
+                        "language": "en-US",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                },
+                "interpret": {
+                    "interpretText": {
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+Add both the `amplifyconfiguration.json` and the `awsconfiguration.json` to your project inside the `res/raw` directory.
```

Comment:
I don't see `awsconfiguration.json` referenced anywhere but here.

Reply:
I didn't see it in iOS doc either, which I found somewhat confusing. I was hoping to add some template, but it seems that there are a lot of things being added to `awsconfiguration.json` for auth and I don't know the current state of auth category. Are more things going to be added to `awsconfiguration.json`? Or less? Are things going to be added to `amplifyconfiguration.json` too on top of what we already have on `awsconfiguration.json`? 

Thinking about ^ gave me a headache, so I just went with keeping silent like iOS did haha

Reply:
We shouldn't need `awsconfiguration.json` at all, AFAIK.

Reply:
removed reference to awsconfiguration